src/roadmap/release-checklist.md

```diff
@@ -104,15 +104,15 @@ version of our release for testing. Here is the current process to do so:
         cp -r workflow-dev-e2e workflow-$WORKFLOW_RELEASE_SHORT-e2e
         cp -r router-dev router-$WORKFLOW_RELEASE_SHORT
 
-  4. Stage copies of all files needing release updates into the appropriate chart directories created above, supplying `--ref release-$WORKFLOW_RELEASE` to specify this branch/ref for lookup of latest commit shas (informing `generate_params.toml`), as well as `--stagingDir <appropriate staging dir>` to inform `deisrel` where to put updated files:
+  4. Run `deisrel git shas --ref=master` to get the current SHAs at the tip of the `master` branch on each repo
```

Comment:
@arschles curious as to the change from release branch (`release-$WORKFLOW_RELEASE`) to `master` as the `--ref` supplied?  (Release branches created in step 2.)